docs/content/preview/manage/backup-restore/snapshots-ycql.md

```diff
@@ -109,49 +104,43 @@ To move a snapshot to external storage, gather all the relevant files from all t
 
     ```sh
-    ./bin/yb-admin export_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e my_keyspace.snapshot
+    ./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> export_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e my_keyspace.snapshot
     ```
 
 1. Copy the newly created snapshot metadata file (`my_keyspace.snapshot`) to the external storage.
 
-1. Copy the data files for all the tablets. The file path structure is:
+1. Copy the data files for all the tablets, as per the following file path structure:
 
-    ```
+    ```path
     <yb_data_dir>/node-<node_number>/disk-<disk_number>/yb-data/tserver/data/rocksdb/table-<table_id>/[tablet-<tablet_id>.snapshots]/<snapshot_id>
     ```
 
-    * `<yb_data_dir>` is the directory where YugabyteDB data is stored. (default=`~/yugabyte-data`)
-    * `<node_number>` is used when multiple nodes are running on the same server (for testing, QA, and development). The default value is `1`.
-    * `<disk_number>` when running yugabyte on multiple disks with the `--fs_data_dirs` flag. The default value is `1`.
-    * `<table_id>` is the UUID of the table. You can get it from the `http://<yb-master-ip>:7000/tables` url in the Admin UI.
-    * `<tablet_id>` in each table there is a list of tablets. Each tablet has a `<tablet_id>.snapshots` directory that you need to copy.
-    * `<snapshot_id>` there is a directory for each snapshot since you can have multiple completed snapshots on each server.
-
-    This directory structure is specific to `yb-ctl`, which is a local testing tool.
-    In practice, for each server, you will use the `--fs_data_dirs` flag, which is a comma-separated list of paths where to put the data (normally different paths should be on different disks).
-    In this `yb-ctl` example, these are the full paths up to the `disk-x`.
+    * *<yb_data_dir>* - a directory where YugabyteDB data is stored. The default value is `~/yugabyte-data`.
+    * *<node_number>* - used when multiple nodes are running on the same server (for testing, QA, and development). The default value is `1`.
+    * *<disk_number>* - used when running YugabyteDB on multiple disks with the `--fs_data_dirs` flag. The default value is `1`.
+    * *<table_id>* - the UUID of the table. You can obtain it from the `http://<yb-master-ip>:7000/tables` URL in the Admin UI.
+    * *<tablet_id>* - each table contains a list of tablets. Each tablet has a `<tablet_id>.snapshots` directory that you need to copy.
+    * *<snapshot_id>* - there is a directory for each snapshot, since you can have multiple completed snapshots on each server.
 
-    {{< note >}}
+    This directory structure is specific to `yb-ctl`, which is a local testing tool. In practice, for each server, you would use the `--fs_data_dirs` flag, which is a comma-separated list of paths for the data. It is recommended to have different paths on separate disks. In the `yb-ctl` example, these are the full paths up to the `disk-x`.
```

Comment:
Does "disk-x" refer to disk-number? It's not in the list

Reply:
It's disk-number. Fixed